gio/src/subprocess_launcher.rs

```diff
@@ -30,30 +30,60 @@ impl SubprocessLauncher {
         }
     }
 
+    #[cfg(any(unix, docsrs))]
+    #[cfg_attr(docsrs, doc(cfg(unix)))]
+    #[doc(alias = "g_subprocess_launcher_take_fd")]
+    pub fn take_owned_fd(&self, source_fd: OwnedFd, target_fd: OwnedFd) {
```

Comment:
This etc should also be `Option`, see above

Reply:
Done, but I am not sure if the change I made should have been done in the way I did (rather than baking the option into `fn unsafe take_fd` itself)

Reply:
Yes I think that would be better

Reply:
This still applies. `None` should map to `-1`